src/AddOn.Episerver.Settings/Core/DefaultSettingsResolver.cs

```diff
@@ -0,0 +1,38 @@
+﻿using EPiServer;
+using EPiServer.Core;
+
+namespace AddOn.Episerver.Settings.Core;
+
```

Comment:
Should this be more specifically named to indicate what it does. DefaultPropertyNameNamingConventionResolver or something more abbreviated?

Reply:
It's now called PropertyNameSettingsResolver

I don't think we need to be fantastically descriptive since it's automagically added, so not many developers will need use it actively.